package.json

```diff
@@ -10,6 +10,26 @@
   "workspaces": [
     "packages/**"
   ],
+  "release": {
+    "branches": [
+      "master",
+      "next"
+    ],
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/changelog",
+      [
+        "@semantic-release/git",
+        {
+          "assets": [
+            "docs"
+          ],
+          "message": "chore(release docs): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
```

Comment:
Do we want it to skip CI, or should this be removed? We set message like this in the stacks wallet specifically to remove `[skip ci]'

Reply:
The current message is the default one I believe. So if the default is desired, this `message` property can be removed, but it will skip CI on the subsequent push like Kyran said.

Reply:
Removed `message ` property as mentioned.